lib/service/cfg.go

```diff
@@ -807,6 +807,10 @@ func (d *Database) CheckAndSetDefaults() error {
 		if err != nil {
 			return trace.BadParameter("invalid Redis database %q address: %q, error: %v", d.Name, d.URI, err)
 		}
+	} else if d.Protocol == defaults.ProtocolSnowflake {
+		if !strings.Contains(d.URI, "snowflakecomputing.com") {
```

Comment:
move the "snowflakecomputing.com"  to constant.

Reply:
Done